AUTHORS

```diff
@@ -1128,3 +1128,10 @@ Leo Battle <leowbattle@gmail.com>
 Advait <apote2050@gmail.com>
 Anurag Bhat <bhat.1@iitj.ac.in>
 Jeremy Monat <jemonat@calalum.org>
+NikhilSDate <47920034+NikhilSDate@users.noreply.github.com>
+Diane Tchuindjo <dtchuindjo@gmail.com>
+Tom Fryers <61272761+TomFryers@users.noreply.github.com>
+Zouhair <zouhair.mahboubi@gmail.com>
+Yurii Kukhatchuk <ykukharchuk@brainstorminc.com>
+YURII KUKHARCHUK <34479303+YKJumper@users.noreply.github.com>
+Yurii Kukharchuk <ykuharchuk@gmail.com>
```

Comment:
The difficulty you are having with the AUTHORS file is because you have changed it. I presume that you have changed it by running `bin/mailmap_check.py --update-authors` but you are not supposed to use the `--update-authors` option as it says in the help:
```
$ bin/mailmap_check.py --help
usage: mailmap_check.py [-h] [--update-authors]

Update the .mailmap and/or AUTHORS files

optional arguments:
  -h, --help        show this help message and exit
  --update-authors  Also update the AUTHORS file. Note that it should only necessary for the
                    release manager to do this as part of the release process for SymPy.
```
Maybe that message should be made clearer: you should not change the AUTHORS file. You should not use the `--update-authors` flag.

To fix this you just need to undo the changes to the AUTHORS file. All the lines I've quoted above should simply be deleted.

Reply:
I have reverted AUTHORS to its previous state.

Reply:
That's step 1. the next step is to make changes to .mailmap.

Reply:
PS D:\Python\sympy> bin/mailmap_check.py
The mailmap file was reordered

For instructions on updating the .mailmap file see:
https://github.com/sympy/sympy/wiki/Development-workflow#update-
mailmap

The following authors will be added to the AUTHORS file at the time of
the next SymPy release.

        Yurii Kukharchuk <ykuharchuk@gmail.com>
PS D:\Python\sympy>